eng/Version.Details.xml

```diff
@@ -35,37 +35,37 @@
     <Dependency Name="Microsoft.WindowsAppSDK" Version="0.0.1">
       <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal</Uri>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Authorization" Version="6.0.4">
-      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
-      <Sha>f9ae0f5d30be2de3c0de61b5673bd8873231d70a</Sha>
+    <Dependency Name="Microsoft.AspNetCore.Authorization" Version="6.0.5">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
```

Comment:
Any preference between azdo and github?

fyi/ @dotnet/aspnet-build of new dependency flow (dotnet/aspnetcore -> dotnet/maui). I'll update our internal queries for build-ops tracking.

Reply:
This auto-updates to whatever repo produced the build we're consuming (in this case, and for most/all public dependency flow, that'll be the github repo) - you don't have to/shouldn't edit it manually